src/clojure_tictactoe/game/players.clj

```diff
@@ -1,21 +1,40 @@
 (ns clojure-tictactoe.game.players
-  (:require [clojure-tictactoe.game.computer-opponent :as ai]
+  (:require [clojure.string :as string]
+            [clojure-tictactoe.game.computer-opponent :as ai]
             [clojure-tictactoe.cli.input.input-getter :as input-getter]))
 
-(def player-atom (atom "x"))
+(defn acceptable-marker-option? [potential-marker]
+  (-> potential-marker
+      (count)
+      (= 1)))
 
-(defn switch-player [current-player]
-  (if (= current-player "x")
-    "o"
-    "x"))
+(defn distinct-markers? [markers]
+  (->> markers
+       (apply =)
+       (not)))
 
-(defn player-swap []
-  (swap! player-atom switch-player))
+(defn acquire-one-marker [player]
+  (let [marker (input-getter/get-player-marker player)]
+    (if (acceptable-marker-option? marker)
+      marker
+      (recur player))))
 
-(defn human-player-move [board]
-  (input-getter/get-player-move board))
+(defn acquire-both-markers [players]
+  (let [markers (map acquire-one-marker players)]
+    (if (distinct-markers? markers)
+      (into [] markers)
+      (recur players))))
```

Comment:
Why is this function recurring? I'm assuming because it's waiting to receive two valid markers. If that's the case, then this is doing more than one thing. Whatever is responsible for handling the validation/user input, should be the thing to call recur (you input getter). This namespace shouldn't be responsible for whether or not both markers were received.